garnetred/loc-collective
# Switch to hooks

#### What's this PR do?
This PR updates all class components to functional and implements hooks. Since I originally began working on this project, there have been changes in norms regarding class vs. functional components, so I thought it was time to update this application. 

#### Where should the reviewer start?


#### How should this be manually tested?
You can run the application locally and verify that everything works correctly or view the demo. 

#### Any background context you want to provide?
The application no longer works on Netlify due to changes to Cors Anywhere, which I was using to bypass potential cors issues. That will likely be the next issue that I address.

#### What are the relevant tickets?

#### Screenshots (if appropriate)

#### Questions
